protos/flow.proto

```diff
@@ -114,10 +101,12 @@ message SyncFlowOptions {
   int32 batch_size = 1;
   map<uint32, RelationMessage> relation_message_mapping = 2;
   int64 idle_timeout_seconds = 3;
+  map<uint32, string> src_table_id_name_mapping = 4;
+  map<string, TableSchema> table_name_schema_mapping = 5;
 }
 
 message NormalizeFlowOptions {
-  int32 batch_size = 1;
+  map<string, TableSchema> table_name_schema_mapping = 1;
 }
 
 message LastSyncState {
```

Comment:
can get rid of `LastSyncState` we only use it to pass around `int64`

Reply:
better to have it around in case we add more foelds and make it backwards compatible

Reply:
Can remove `last_synced_at`

Reply:
removed `last_synced_at`